### PR TITLE
highlight `values.yaml.tmpl` as yaml

### PR DESCRIPTION
**Pull request by kaworu**
To ease reviewing diff on the template file.

Screenshot 2022-06-21 at 10-52-54 highlight `values yaml tmpl` as yaml by kaworu · Pull Request #20250 · cilium_cilium

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/install/kubernetes/cilium/values.yaml.tmpl linguist-language=yml
```
Comment (sayboras) [inline .gitattributes:1]: nice I didn't know it's possible, I manually associate file extension in Goland :1st_place_medal:

**Review by sayboras — APPROVED** [review]
:shipit:

File: .gitattributes (hunk 1 above)
Comment (kaworu) [reply .gitattributes:1]: FWIW I tested it with a dummy commit and it was highlighted as expected in the PR submit preview diff.

**Review by joestringer — APPROVED** [review]
Thanks, I had no idea about this :)

**Review by sayboras — APPROVED** [review]

**Pull request merged**